src/components/Button.js

```diff
@@ -2,16 +2,31 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import './Button.css';
 
-function Button({ buttonName }) {
+function Button({ buttonName, wide, color }) {
+  const styles = wide
+    ? { width: '50%' }
+    : { width: '25%' };
+
+  styles.backgroundColor = color
+    ? '#dfdfdf'
+    : '#F29444';
+
   return (
-    <button type="button" className="Button" id="button">
-      {buttonName}
-    </button>
+    <div className="button" style={styles} id="button">
```

Comment:
- [ ] **[OPTIONAL]** It's better to depend on classes instead of using inline styles for your app styling. This is also recommended by React  styling [docs](https://reactjs.org/docs/dom-elements.html#style).

A better approach is that you create ready styles in your css file and then append it to `className` conditionally to the element, depending on the prop values.

[styling docs](https://reactjs.org/docs/faq-styling.html)